sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -135,6 +135,14 @@ abstract class JdbcDialect extends Serializable with Logging {
     s""""$colName""""
   }
 
+  /**
+   * Quotes the field attribute. This is used to put quotes around the field in case
+   * the attribute contains characters that require quotes (e.g. space).
+   */
+  def quoteFieldAttribute(attribute: String): String = {
+    attribute
+  }
+
   /**
```

Comment:
Maybe we need this method to let subclasses implement quota field attribute when it contains spaces.